eden/modifier/graph/cycle.py

```diff
@@ -0,0 +1,208 @@
+import eden
+from collections import defaultdict
+
+
+class Annotator:
+    def fit(self):
+        pass
```

Comment:
want to raise an https://docs.python.org/3/library/exceptions.html?highlight=notimplementederror#NotImplementedError error?